subworkflows/nf-core/fasta_build_add_kraken2/main.nf

```diff
@@ -0,0 +1,35 @@
+include { KRAKEN2_ADD   } from '../../../modules/nf-core/kraken2/add/main'
+include { KRAKEN2_BUILD } from '../../../modules/nf-core/kraken2/build/main'
+
+workflow FASTA_BUILD_ADD_KRAKEN2 {
+
+    take:
+    ch_fasta              // channel: [ val(meta), fasta ]
+    ch_taxonomy_names     // channel: [ names.dmp ]
+    ch_taxonomy_nodes     // channel: [ nodes.dmp ]
+    ch_accession2taxid    // channel: [ acc2taxidfile ]
+    val_cleanintermediate // value: [ true | false ]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    ch_fastas_for_kraken2add = ch_fasta
+                                .map {
+                                    meta, fasta ->
+
+                                    [[id: 'db'], fasta]
+                                }
+                                .groupTuple()
```

Comment:
```suggestion
    ch_fastas_for_kraken2add = ch_fasta.map { meta, fasta -> [[id: 'db'], fasta]}.groupTuple()
```
I do prefer code to be a bit more compact, but that's more a style question, this one might be a bit extreme, but you're definitively on the other side of the curve

Reply:
OK this one would be OK to collapse into one, but indeed I normally prefer one function per line